src/Utilities/MakeArray.hpp

```diff
@@ -28,8 +28,15 @@ namespace MakeArray_detail {
 /// is no risk of copy-after-move
 template <std::size_t size, typename T, std::size_t... indexes>
 SPECTRE_ALWAYS_INLINE constexpr std::array<std::decay_t<T>, size>
-make_array_impl(T&& value,
-                std::integer_sequence<size_t, indexes...> /* unused */) {
+make_array_impl(
```

Comment:
Use `// clang-format off` and `// clang-format on`, then format this to something sane